components/dash-core-components/tests/integration/dropdown/test_dropdown_radioitems_checklist_option_title_prop.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: UTF-8 -*-
+
+import pytest
+from dash.testing import wait
+from dash import Dash, Input, Output, dcc, html
+
+
+@pytest.mark.DCC793
+@pytest.mark.parametrize(
```

Comment:
Non blocking, parametrize is slow to run because it start/stop both the server and the browser. I think the test could just add all the components to the layout with different ids and performs the assertions statically.
For further refactors the callback is not really needed for testing purpose and the props can be given directly in the layout with a static title.